### ISSUE TITLE
Product status is reset after each import in Magento Cloud

### ISSUE THREAD

**Issue by yutv** [akeneo/magento2-connector-community#200]
### Preconditions (*)
1. Magento Cloud 2.3.4
2. Akeneo Connector 100.4.5
3. All products are enabled in Akeneo

### Steps to reproduce (*)
1. Go to  `Stores > Configuration > Catalog > Akeneo Connector > Products` and set 
    **Default product status** = Enabled
2. Import Products from Akeneo
3. Pick a product in Magento and disable it.
4. Import Products from Akeneo

### Expected result (*)
1. The product should remain disabled

### Actual result (*)
1. The product is enabled

### Code

https://github.com/akeneo/magento2-connector-community/blob/1e87e2f0214345f89db4aa2cfa1226c8dafec178/Job/Product.php#L1363
The `$identifierColumn` = `'row_id'` not `'entity_id'` in Magento Clound. Also the `catalog_product_entity.entity_id` field may be not equal to the `catalog_product_entity.row_id` one. As result the SQL query from the code mentioned above will return empty/wrong result.

The additional JOIN expression to the `catalog_product_entity` table should be added here like shown below:
```sql
SELECT `a`.`_entity_id` AS `entity_id`,
       `a`.`_is_new`,
       `b`.*
FROM `tmp_akeneo_connector_entities_product` AS `a`
INNER JOIN `catalog_product_entity` AS `e` ON e.entity_id = a._entity_id
INNER JOIN `catalog_product_entity_int` AS `b` ON b.row_id = e.row_id
WHERE (a._is_new = 0)
  AND (a._status = 1)
  AND (b.attribute_id = '97')
```
Current SQL query is:
```sql
SELECT `a`.`_entity_id` AS `entity_id`,
       `a`.`_is_new`,
       `b`.*
FROM `tmp_akeneo_connector_entities_product` AS `a`
INNER JOIN `catalog_product_entity_int` AS `b` ON a._entity_id = b.row_id
WHERE (a._is_new = 0)
  AND (a._status = 1)
  AND (b.attribute_id = '97')
```

**Comment by Dnd-Gimix** [comment]
Hello @yutv ,

Thank you for your interest in Akeneo Connector.

We have provided a bug fix for this issue in the version 100.4.7 of the connector : https://github.com/akeneo/magento2-connector-community/releases/tag/v100.4.7

Please tell us if it solved your issue.

Regards,

**Comment by yutv** [comment]
Hello @Dnd-Gimix, yes the issue is solved. Thanks.
